### DISCUSSION TITLE
About exp options

### DISCUSSION THREAD

**Discussion by dragonknightksr** [YimMenu/HorseMenu#177] [Q&A] [answered]
Hello! The menu is art of work! I just have a question, is give EXP options in Recovery tab safe to use?

Many thanks!

**Comment by tuyilmaz** [comment]
So as the person who added them I've been using that method for over 8-9 months and it seems safe for now. But ofcourse to be safer don't abuse them and make it obvious.

**Reply by dragonknightksr** [reply]
Ty sir!

**Comment by Rxann** [comment, marked as answer]
Yes they are safe, but don't abuse them. It is up to you to determine an amount of EXP that you are comfortable with. We are not responsible.
